### DISCUSSION TITLE
[Jetbrains plugin] In "Find in all files" copilot does search for completions but does not display them

### DISCUSSION THREAD

**Discussion by cmp-nct** [community/community#36061] [Copilot Conversations]
When using "Find in all files" in PHPStorm the IDE is quite powerful, you can work in it, get completions and use most of the functionality while traversing multiple files not currently "open" in the editor.
The CoPilot plugin also searches for completions when you work in there but it can not display anything in that window.
So you only see that it was working on it, that it found a completion but you couldn't use it.

It's a minor bug. Short term it could just switch off when you work in that window. Longterm it should work there too.

**Comment by iamcjwilliams** [comment]
@comp-nect Thank you for your feedback! Our copilot team is investigating this
